knative-operator/deploy/resources/console_cli_download_kn_deployment.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: serverless-operator
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Should this be a Knative Service instead of a Deployment? So that it scales to 0 when not in use?